### PR TITLE
UIP-1046: Add responsive classes

### PR DESCRIPTION
**Pull request by reywright**
## Checklist

- [ ] Docs have been rebuilt (`yarn build:full`)
- [ ] All unit tests pass
- [ ] Snapshots have been updated
- [ ] No lint errors or warnings have been introduced
- [ ] **[Version bumped](https://github.com/cbinsights/form-design-system#updating-version-number) if appropriate**

### CODE REVIEW

**Comment by reywright** [comment]
There's some errant changes in here from prettier... not sure why.

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 
 ---
```
Comment (reywright) [inline CHANGELOG.md:32]: Needs changelog + minor update before merging.

File: src/base-styles/helperClasses/index.stories.mdx [hunk 2]
```diff
@@ -20,6 +20,44 @@ export const positionStyles = {
...
+
+Example of usage:
+
+```
+<div className="padding--top--l lg:padding--top--xl" />
+```
+
+Guidelines / Notes:
+
+- Don't use `sm:` to target mobile devices
```
Comment (akdetrick) [inline src/base-styles/helperClasses/index.stories.mdx:57]:
```suggestion
- Don't use `s:` to target mobile devices
```

File: src/base-styles/helperClasses/index.stories.mdx [hunk 3]
```diff
@@ -20,6 +20,44 @@ export const positionStyles = {
...
+
+- [margin & padding](/?path=/docs/style-utilities-style-utilities--margin-and-padding)
+- [align](?path=/docs/style-utilities-style-utilities--text-align)
+- [alignChild](?path=/docs/style-utilities-style-utilities--child-align)
+- [display](/?path=/docs/style-utilities-style-utilities--display)
+
+Example of usage:
+
+```
+<div className="padding--top--l lg:padding--top--xl" />
```
Comment (akdetrick) [inline src/base-styles/helperClasses/index.stories.mdx:52]:
```suggestion
<div className="padding--top--l l:padding--top--xl" />
```

File: src/base-styles/helperClasses/position/align/viewports.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+.align--left {
+  text-align: left;
+}
+.align--center {
+  text-align: center;
+}
+.align--right {
+  text-align: right;
+}
```
Comment (akdetrick) [inline src/base-styles/helperClasses/position/align/viewports.css:9]: Do these belong in `align/base.css`?

File: src/base-styles/helperClasses/index.stories.mdx (hunk 2 above)
Comment (akdetrick) [reply src/base-styles/helperClasses/index.stories.mdx:57]: This point might be a little misleading as it implies you can target viewports based on max-width, which you can't

Comment (reywright) [reply src/base-styles/helperClasses/index.stories.mdx]: Screen Shot 2021-04-06 at 1 59 02 PM

Would it help if we added some of the context in this picture that inspired this line?

**Review by akdetrick — CHANGES_REQUESTED** [review]
(see comment above) Just need to resolve the question of `align/viewports.css` before merging. Otherwise looks good to go

**Pull request merged**